src/core-sdk-tasks/AddMetadataIsPE.cs

```diff
@@ -26,16 +26,9 @@ public override bool Execute()
             {
                 var resultItem = new TaskItem(item);
                 item.CopyMetadataTo(resultItem);
-                
-                if (File.Exists(resultItem.GetMetadata("FullPath")) &&
-                    HasMetadata(resultItem.GetMetadata("FullPath")))
-                {
-                    resultItem.SetMetadata("IsPE", "True");
-                }
-                else
-                {
-                    resultItem.SetMetadata("IsPE", "False");
-                }
+
+                var isPe = File.Exists(resultItem.GetMetadata("FullPath")) && HasMetadata(resultItem.GetMetadata("FullPath"));
+                resultItem.SetMetadata("IsPE", isPe.ToString());
```

Comment:
> [!WARNING]  
> This is just a rant. It doesn't directly relate to this PR but uses this simple change as an example.

Perfect example of why I don't like if-else statements. Both bodies (true body and false body) are identical statements. The only difference is the data being passed to the method. The data itself is a literal value of the predicate result. Thus, I just make a single statement to the method and inject the predicate result directly. The conversion of a boolean to string is built into the boolean type. No need to reinvent the wheel.

I see this same pattern happen on a larger scale and the only way to recognize that it happens is to refactor and simplify. This also removes branching logic since it takes the branches and collapses them into data, not statements. As far as I'm concerned, the language doesn't allow a more straightforward way to write this.